crates/weaver_semconv_gen/Cargo.toml

```diff
@@ -19,11 +19,11 @@ similar = "2.4.0"
 nom = "7.1.3"
 
 thiserror.workspace = true
-serde.workspace = true
-serde_yaml.workspace = true
-serde_json.workspace = true
-rayon.workspace = true
-walkdir.workspace = true
+#serde.workspace = true
```

Comment:
Why are all of these commented out?

Reply:
Unless I am mistaken, these dependencies were not used in `weaver_semconv_gen`. I believe I saw this in the output of `cargo machete`.

Reply:
It's odd, when I try to build/use this crate locally it fails unless they're included.  Likely my setup is messed up, but this is why they're in there.

Reply:
You can remove the comments though, if we need them we'll add them back in fully.

Reply:
Checked and Done